styles/components.html

```diff
@@ -0,0 +1,388 @@
+<html>
```

Comment:
I don't think a *.html file should be in styles order. WDYT?

Reply:
moving to new directory